packages/@okta/vuepress-theme-prose/global-components/SignUp.vue

```diff
@@ -446,8 +589,39 @@ export default {
     this.analyticsValues = getAnalyticsValues();
   },
   methods: {
+    setHeight(isNotUSAAndCanada) {
+      let height;
+      if (window.innerWidth > 1314) {
+        if (!isNotUSAAndCanada) {
+          setTimeout(function() {
+            height = document.querySelector('.signup__form').getBoundingClientRect().height + 139 + 'px';
+            document.querySelector('.signup__items.active').style.height = height;
+          }, 10);
+        } else {
+          document.querySelector('.signup__items.active').style.height = 'auto';
+        }
+      }
+      window.onresize = function() {
+        if (window.innerWidth < 1314) {
+          document.querySelector('.signup__items.active').style.height = 'auto';
+        } else {
+          document.querySelector('.signup__items.active').style.height = height;
+        }
+      }
+    },
+    hideForm() {
+      if (document.querySelector('.signup__items.active') && !this.formHidden) {
+        this.setHeight(true);
+      }
+      this.formHidden = !this.formHidden;
+      if (!this.formHidden) {
+        this.setHeight(true);
+      } else {
+        this.setHeight();
+      }
+    },
     getTheme: function() {
-      return JSON.parse(storage.getItem(THEME_MODE_KEY)) === true ? "dark" : "light";
+      return document.querySelector('body').classList.contains('dark-theme') ? 'dark' : 'light'
```

Comment:
Why are we querying the body classes to infer the theme rather than using the storage value?

Reply:
changed to localStorage